ledger/accountdb.go

```diff
@@ -132,7 +132,7 @@ var createResourcesTable = []string{
 		PRIMARY KEY (addrid, aidx) ) WITHOUT ROWID`,
 }
 
-var createBoxTable = []string{
+var kvStoreTable = []string{
```

Comment:
Agree with getting rid of "box", but similar variables use naming like `createKvStoreTable`

Reply:
@jannotti Sorry - I missed the existing convention.  I think it's fixed in https://github.com/algorand/go-algorand/pull/4818/commits/0ea6c872974eb6425ac910aacb17cd3d70089b4d.